File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag50/ExercicioE.java
package manzanoPag50;

import java.util.Scanner;

public class ExercicioE {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int contadora = 0;
		int valor = 0;
		int resultado = 0;
		int guardaValor = 0;
		int antecessor = 0;
		int fatorial = 0;

		do {
			System.out.println("Digite um valor para saber seu fatorial:");
			valor = sc.nextInt();

			antecessor = valor;
			fatorial = valor;

			do {

				antecessor = antecessor - 1;
				fatorial = fatorial * antecessor;
				resultado = fatorial;
				System.out.println(resultado);

			} while (antecessor > 2);

			guardaValor = guardaValor + resultado;

			contadora = contadora + 1;
		} while (contadora < 14);

		System.out.println("A soma do fatorial de todos os numeros digitado é " + guardaValor);

		sc.close();

	}

}

File: ExerciciosJavaScript(LógicadeProgramacao)/exercicios pag 50(Manzano)/js/exercicioE.js

/* Elaborar um programa que efetue a leitura de 15 valores numéricos inteiros e no final apresente o 
        total do somatório da fatorial de cada valor lido.  */


        var contadora = 0
        var valor = 0
        var resultado = 0
        var guardaValor = 0
        var antecessor = 0
        var fatorial = 0


    do {
         valor = parseInt(prompt("Digite um valor para saber seu fatorial:"))

         antecessor = valor
         fatorial = valor

         do {

         antecessor = antecessor - 1
         fatorial = fatorial * antecessor
         resultado = fatorial
         alert(resultado)

         } while (antecessor > 2)
         
         guardaValor = guardaValor + resultado


         contadora = contadora + 1
    } while (contadora<14)
   
   alert("A soma do fatorial de todos os numeros digitado é "+ guardaValor)